src/Administration/Resources/app/administration/src/module/sw-property/page/sw-property-list/index.js

```diff
@@ -71,9 +72,16 @@ export default {
         onConfirmDelete(id) {
             this.showDeleteModal = false;
 
-            return this.propertyRepository.delete(id).then(() => {
-                this.getList();
-            });
+            return this.propertyRepository
+                .delete(id)
+                .then(() => {
+                    this.getList();
+                })
+                .catch(() => {
+                    this.createNotificationError({
```

Comment:
There are two approach:

1. please catch if the exception match the pattern "Cannot delete or update a parent row: a foreign key constraint fails" and show this error message
2. Check if the property is used in at least one of product `options` or `properties`

```
criteria.addFilter(Shopware.Data.Criteria.equals('options.groupId', '<deleting_id>'));

// also check for this case
criteria.addFilter(Shopware.Data.Criteria.equals('properties.groupId', '<deleting_id>'));

criteria.setTotalCountMode(0);
criteria.setLimit(1);

productRepository.searchIds(criteria);
```